edmq_backend/src/scripts/shazam_data.py

```diff
@@ -0,0 +1,22 @@
+import requests
+import json
+
+# Monthly quota is 500 from RapidAPI
+shazam_headers = {
+    'x-rapidapi-key': "194d3fca7emsh5f18f2dac01d040p176829jsnf6975cc79831",
+    'x-rapidapi-host': "shazam.p.rapidapi.com"
+    }
+shazam_url = "https://shazam.p.rapidapi.com/search"
+
+def get_title_artist(term):
+    d = {}
```

Comment:
what is d?

Reply:
literally just dictionary. I will update to specify what I'm actually returning, maybe title_artist_dict

Reply:
i like that better